tasks/task109_SMSspamcollection_SpamSMSdetection.json

```diff
@@ -0,0 +1,11545 @@
+{
+    "Contributors": [
+        "Savan Doshi"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Source": [
+        "SMS Spam Collection v.1"
+    ],
+    "Definition": "In this task, you're given Text messages and you need to classify that message into Spam messages or ham (legitimate) messages based on the content of the text message.",

```

Comment:
We need to reformulate this description in a way that: 
 - it defines what input constitutes "spam"? Let's provide a definition. If this is a crowdsourced dataset, we should use the definitions used in their task. 
 - What is "ham"? if it is the opposite of "spam", we can call it "not spam"?
 - The definition should be explicit what we expect as the output.

Reply:
I have made the changes in the JASON.

Yes we can call it "not spam", however in the dataset it is called as "ham".